ci/benchmark_ai.sh

```diff
@@ -0,0 +1,218 @@
+#!/bin/bash
+
+#Get option to run test on GPU/CPU image
+while getopts "g:" option; do
+  case $option in
+    g)      use_gpu=$OPTARG;;
+  esac
+done
+echo $use_gpu
+
+report=report.csv
```

Comment:
Its better to move globals to the top. That way we don't mistake them for a command.